receiver/awscontainerinsightreceiver/internal/host/utilconst_windows.go

```diff
@@ -0,0 +1,14 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !linux
+// +build !linux
+
+package host
+
+// These variables are invalid for Windows
+const (
```

Comment:
Another nit: This works, but could there be other more readable (and maintainable) ways to refactor this and enable the ability of having OS-specific behaviour or properties? Not sure how the codebase is what might work best so would rely on your good judgement.

Reply:
I saw similar pattern of defining OS specific const in repo.